javascripts/discourse-kanban/services/kanban-helper.js.es6

```diff
@@ -101,7 +101,7 @@ export default Ember.Service.extend({
           if (param) {
             var categories = param
               .split(",")
-              .map(c => Discourse.Category.findBySlug(c));
+              .map(c => Discourse.Category.findBySlug(...c.split('/')));
```

Comment:
The parameters to findBySlug are `(category, parentCategory)`, so I think the array will need to be reversed here?

Reply:
I was thinking we'd just make the format of the param ``child/parent``, but yeah, it'd be better to make it the more standard ``parent/child``